src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(number) {
+      calculator.result += number;
+    },
+
+    subtract(number) {
+      calculator.result -= number;
+    },
+
+    multiply(number) {
+      calculator.result *= number;
+    },
+
+    divide(number) {
+      calculator.result /= number;
+    },
+
+    operate(callback, number) {
+      callback(number);
+
+      return this;
```

Comment:
You could add `calculator.result` as a second parameter to your callback function in `operate method`,
so you don't have to change `calculator.result` in every operating method
Like this:
```
    divide(result, number) {
      return result / number;
    },

  operate(callback, number) {
        this.result = callback(this.result, number);
```
        
I'm not sure about naming of variables but hope that you could get the idea.

Reply:
agree, fix @tania-boiar comment
methods for working with numbers shouldn't know anything about some object or some property
they just should accept 2 numbers and provide needed result